### ISSUE TITLE
XXX user joined channel -messages errors on slackbridge

### ISSUE THREAD

**Issue by olmari** [matrix-org/matrix-appservice-slack#179]
A matrix-room that is bridged to multiple places, when (at least) IRC-user joins to room, matrix-slack bridge generates debug error:

```
Aug 11 21:24:40 matrix node[30329]: Aug-11 21:24:40.165 INFO bridge ::ffff:127.0.0.1 - - [11/Aug/2019:18:24:40 +0000] "PUT /transactions/7724?access_token=<REDACTED> HTTP/1.1" 200 2 "-" "Synapse/1.2.1"
Aug 11 21:24:40 matrix node[30329]: Aug-11 21:24:40.168 DEBUG substitutions Event from matrix: {"age":613,"content":{"avatar_url":null,"displayname":"Erraverunt","membership":"join","body":""},"event_id":"$15655478792747646tiHnr:matrix.org","origin_server_ts":1565547879543,"prev_content":{"membership":"leave"},"replaces_state":"$15653915401787561YqNgQ:matrix.org","room_id":"!MjVdMmsFAxVzHUZLiV:matrix.org","sender":"@freenode_Erraverunt:matrix.org","state_key":"@freenode_Erraverunt:matrix.org","type":"m.room.member","unsigned":{"age":613,"prev_content":{"membership":"leave"},"prev_sender":"@freenode_Erraverunt:matrix.org","replaces_state":"$15653915401787561YqNgQ:matrix.org"},"user_id":"@freenode_Erraverunt:matrix.org"}
Aug 11 21:24:40 matrix node[30329]: Aug-11 21:24:40.172 INFO bridge [-] POST http://localhost:8008/_matrix/client/r0/join/!MjVdMmsFAxVzHUZLiV%3Amatrix.org (AS) Body: "{}"
Aug 11 21:24:40 matrix node[30329]: Aug-11 21:24:40.224 INFO bridge [-] POST http://localhost:8008/_matrix/client/r0/join/!MjVdMmsFAxVzHUZLiV%3Amatrix.org (AS) HTTP 200 "{\"room_id\": \"!MjVdMmsFAxVzHUZLiV:matrix.org\"}"
Aug 11 21:24:40 matrix node[30329]: Aug-11 21:24:40.225 INFO bridge [-] GET http://localhost:8008/_matrix/client/r0/rooms/!MjVdMmsFAxVzHUZLiV%3Amatrix.org/state (AS) Body:
Aug 11 21:24:40 matrix node[30329]: Aug-11 21:24:40.412 INFO bridge [-] GET http://localhost:8008/_matrix/client/r0/rooms/!MjVdMmsFAxVzHUZLiV%3Amatrix.org/state (AS) HTTP 200 "[{\"content\": {\"avatar_url\": \"mxc://privacytools.io/uLOBIGzpIxwSZImBKfXodZR
Aug 11 21:24:40 matrix node[30329]: Aug-11 21:24:40.624 ERROR BridgedRoom HTTP Error:  { ok: false,
Aug 11 21:24:40 matrix node[30329]:   error: 'no_text',
Aug 11 21:24:40 matrix node[30329]:   warning: 'missing_charset',
Aug 11 21:24:40 matrix node[30329]:   response_metadata: { warnings: [ 'missing_charset' ] } }
```

EDIT: on version commit 48a5a7b9b6e5151ee539e2122f9770b2e173ace6

**Comment by olmari** [comment]
Errors rootcause could potentially also be on matrix(synapse), on appservice-irc, or so on, but something in the flow generates that.

**Comment by Half-Shot** [comment]
This got fixed in #238
